README.md

```diff
@@ -1,12 +1,46 @@
-![](./assets/applicaster.png)
-
 # zapp-plugins-examples
 
 This repostory contains few `Zapp Plugins` examples that will help you to get started with creating different types of plugins for Zapp.
+If you are new with developing plugins for Applicaster Zapp platform, you start with the [Getting Started Guide](https://developer-zapp.applicaster.com/getting-started/zapp-plugins.html).
 
 ## video player plugin
+
 Zapp enables to add different players to the Zapp Marketplace.
-This example include an Andorid and iOS projects that present the implementation of the the `Zapp pluggable player API`.
+This example include an Andorid and iOS projects that present the implementation of the `Zapp pluggable player API`.
+
+* [Full guide for creating a player plugin](https://developer-zapp.applicaster.com/player/intro.html)
+* [iOS example](./VideoPlayer/iOS)
+* [Android example](./VideoPlayer/Android)
+
+## display advertisement plugin
+
+Zapp enables to add different display advertisement plugins to the Zapp Marketplace.
+This example include an Andorid and iOS projects that present the implementation of display advertisement plugin.
+
+* [Full guide for creating a advertisement plugin](https://developer-zapp.applicaster.com/advertisement/intro.html)
+* [iOS example](./Advertisement/iOS)
+* [Android example](./Advertisement/Android)
+
+## analytics plugin
+
+Zapp enables to add different analytics providers to the Zapp Marketplace.
+This example include an Andorid and iOS projects that present the implementation of an analytics plugin.
+
+* [Full guide for creating a advertisement plugin](https://developer-zapp.applicaster.com/analytics/developing_analytics_plugins/developing_analytics_plugins.html)
+* [iOS example](./AnalyticsPlugin/iOS)
+* [Android example](./AnalyticsPlugin/Android)
+
+## login plugin
+
+Zapp enables to add different login providers to the Zapp Marketplace.
+This example include an iOS project that present the implementation of an login plugin.
+
+* [iOS example](./LoginPlugin/iOS)
```

Comment:
Please add link to the docs:
https://developer-zapp.applicaster.com/login/intro.html